oxcaml/testsuite/tools/parsecmm.mly

```diff
@@ -177,7 +177,8 @@ fundecl:
            ]
            else [ Reduce_code_size ];
          fun_poll = Lambda.Default_poll;
-         fun_dbg = debuginfo ()} }
+         fun_dbg = debuginfo ();
+         fun_ret_type = Cmm.typ_val } }
```

Comment:
I think we should probably do the same thing
we do for parameters. I reckon this would lead
to a large-ish diff because the syntax would
have to be tweaked, so this should probably
happen in a follow-up (just leaving a CR in
this pull request).